client/src/components/calendar/CalendarActions.js

```diff
@@ -0,0 +1,55 @@
+import React, {useState, useEffect} from 'react';
+import CalendarSideNav from './CalendarSideNav';
+import CalendarView from './CalendarView';
+import CalendarStyles from '../styles/CalendarStyles';
+
+const CalendarActions = ({currentBoard}) => {
+  const classes = CalendarStyles();
+  const [calendarEvents, setCalendarEvents] = useState();
+  const [draggableEvents, setDraggableEvents] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const loadData = async () => {
+      const cards = [];
+      const draggableCards = [];
+      currentBoard.columns.forEach((columns) => {
+        columns.cards.forEach((cardObject) => {
+          const card = {};
+          card.title = cardObject.name;
+          // eslint-disable-next-line no-param-reassign, no-underscore-dangle
+          card.id = cardObject._id;
```

Comment:
Just gonna leave a note, these ._id issues will fix once we merge in my route refactoring